### ISSUE TITLE
Task Groups

### ISSUE THREAD

**Issue by kbond** [zenstruck/schedule-bundle#40]
See https://github.com/zenstruck/schedule-bundle/issues/39#issuecomment-790636785

**Comment by danie-dejager** [comment]
@kbond Any traction on this improvement?

**Comment by kbond** [comment]
No, I'm kind of winding down active development of this bundle (at least by myself) in favor of `symfony/scheduler`. I plan to maintain it for the foreseeable future (at least until 5.4 is EOL).

**Comment by PhilETaylor** [comment]
I would love to move to `symfony/scheduler` but despite being launched in March 2023, there is still no documentation on how to actually use it and no one has time to reverse engineer it to guess how to use it :) Im still using this bundle for now too. 

Also the `symfony/scheduler`  is flagged as experimental and thus not covered by the BC promise - yet.

**Comment by kbond** [comment]
> there is still no documentation on how to actually use it

I know... This should be coming soon - hopefully a hackday item next week as SymfonyCon.

> Also the symfony/scheduler is flagged as experimental and thus not covered by the BC promise - yet.

Yep, this will be removed in 6.4/7.0.

**Comment by PhilETaylor** [comment]
Absolutely no pressure/rush from me :) just pointing out why people (like me) are still using, and grateful for, this project.
